tools/ova/script/config.sh

```diff
@@ -37,13 +36,18 @@ function format {
 }
 
 #Modify hostname
-ip=$(ip addr show eth0|grep "inet "|tr -s ' '|cut -d ' ' -f 3|cut -d '/' -f 1)
-if [ -n "$ip" ]
+hostname=$(hostname)
```

Comment:
you may want to use "hostname --fqdn"?

And I recall sometimes this command will generate error, if that happens we may want to fallback to using IP to setup, but explicitly log it.